mozilla-release/toolkit/components/extensions/parent/ext-runtime.js

```diff
@@ -139,6 +139,19 @@ this.runtime = class extends ExtensionAPI {
           return Promise.resolve(info);
         },
 
+        getEnterprisePolicy: function() {
+          if (extension.id !== "cliqz@cliqz.com") {
+            return Promise.reject("NO");
+          }
+          const info = {};
+          const getBranches = Services.prefs.getBranch("extensions.cliqz.enterpirse.");
```

Comment:
why not make this an experimenta API in the extension or even simply use`hostPrefs` ? it would be far easier to test, even in Firefox ;)

Reply:
I would love to do this :) 
But for sure we need to discuss this with extension team.